src/routes/createNewHouse.routes.js

```diff
@@ -0,0 +1,14 @@
+import createNewHouseController from '../controllers/createNewHouse.controllers';
+
+const createNewHouseRoute = (router) => {
+    router.post('/create-new-house', async (req, res) => {
+      const newHouse = await createNewHouseController(req.body)
+      try {
```

Comment:
Unfortunately, this `try-catch` block doesn't make much difference as if there is an error in `createNewHouseController` the route will return a response with the status 201 and an error message in its body.